pkg/cloudprovider/providers/azure/azure_loadbalancer.go

```diff
@@ -199,7 +199,8 @@ func (az *Cloud) getServiceLoadBalancer(service *v1.Service, clusterName string,
 
 	// check if the service already has a load balancer
 	if existingLBs != nil {
-		for _, existingLB := range existingLBs {
```

Comment:
`existingLB` will always change in every iteration, what's the orignial issue here?

Reply:
The value changes, but not its address